docker/example.env

```diff
@@ -0,0 +1,77 @@
+## Common Docker configuration
+
+# Service broker
+# SERVICE_BROKER=redis://redis_common:6379/0
+
+# Lisk Service log config
+# SERVICE_LOG_CONSOLE=false
+# SERVICE_LOG_FILE=false
+# SERVICE_LOG_GELF=false
+# SERVICE_LOG_LEVEL=debug
+# SERVICE_LOG_STDOUT=true
+
+# Lisk Service Core
+# SERVICE_CORE_REDIS=redis://redis_core_persistent:6379/0
+# SERVICE_CORE_REDIS_VOLATILE=redis://redis_core_volatile:6379/0
+# SERVICE_CORE_MYSQL=mysql://lisk:password@mysql_core:3306/lisk
+
+# Lisk static assets, ie. known account lists
+LISK_STATIC=https://static-data.lisk.com
+
+# Lisk Service geolocation backend
```

Comment:
Does empty mean disabled?

Reply:
Yes, it is equivalent to that.

Reply:
Maybe the comment should mention that then?

Reply:
Added explanation what empty means.